ncrypt/crypt_gpgme.c

```diff
@@ -3459,12 +3459,10 @@ static const char *crypt_format_str(char *buf, size_t buflen, size_t col, int co
       *p = '\0';
 
       {
-        time_t tt = 0;
-
         if (key->kobj->subkeys && (key->kobj->subkeys->timestamp > 0))
-          tt = key->kobj->subkeys->timestamp;
-
-        tm = mutt_date_localtime(tt);
+          tm = mutt_date_localtime(key->kobj->subkeys->timestamp);
+        else
+          tm = mutt_date_localtime(0); // Default to 1970-01-01
       }
```

Comment:
I believe the brackets surrounding this piece of code aren't needed, but I'm not very familiar with the syntax so wanted to ask you first.

Reply:
Yes, they can go too.
It's a bit ugly, but you can use `{}`s to limit the scope of variables:

Reply:
Thanks :) updated